vkconfig/mainwindow.cpp

```diff
@@ -701,8 +712,8 @@ void MainWindow::EditCustomPathsClicked(ContigurationListItem *item) {
     LoadConfigurationList();  // Force a reload
 }
 
-void MainWindow::toolsSetCustomPaths(bool bChecked) {
-    (void)bChecked;
+void MainWindow::toolsSetCustomPaths(bool checked) {
```

Comment:
Thank you!

Reply:
Why was the second line removed? Is the variable used elsewhere? Otherwise you get a compiler warning about an unused variable.

Reply:
(void)checked is still below, no idea why the change is not visible here...